packages/@react-aria/overlays/src/usePopover.ts

```diff
@@ -113,12 +114,12 @@ export function usePopover(props: AriaPopoverProps, state: OverlayTriggerState):
     isDisabled: isNonModal || !state.isOpen
   });
 
-  useLayoutEffect(() => {
+  useEffect(() => {
```

Comment:
any particular reason for the change?

Reply:
Yeah we need to wait until focus moves into the popover before adding `inert`, otherwise we'll lose focus on the original element.